### ISSUE TITLE
Nominating Zsolt Borbély (@bzsolt) to become an IoT.js committer

### ISSUE THREAD

**Issue by glistening** [Samsung/iotjs#857]
I would like to nominate Zsolt Borbély (@bzsolt) to become an IoT.js committer.

Here are merged PRs:

- `require` keyword in comment: after #642 this trick isn't needed anymore (#807)
- Followup patch for the Jerry-debugger (#805) 
- Activate sanitizer-jobs on Travis CI (#787) 
- Sync test/testsets.json with the existing tests (#790)
- Allow to use JerryScript-debugger (#722) 
- Update JerryScript (#720) 
- Introduce `--jerry-profile` buildoption (#715) 
- Update JerryScript-submodule (#705) 
- Fix wrong references in docs/README.md (#688) 
- Eliminate shadow declarations (#681) 
- Activate field-initializer compiler warning (#675) 
- Enable unused variable check (#667) 
- Add address- and undefined behavior sanitizer to Travis CI (#646) 
- Update function-names in js2c.py after cleanup (#649) 
- Add missing iotjs_module_uart-i686-linux.c (#643) 
- Update JerryScript's urls (#608) 
- Fix i2c.writeBytes() and remove the duplicate of I2C_SMBUS_BLOCK_MAX (#598)
- Update zlib dependency to 1.2.11 (#599)
- Put dependencies(Linux) into a file and add an ARM-build to Travis CI 
- Pass 'target_board' option to libtuv

**Comment by LaszloLango** [comment]
+1

**Comment by yichoi** [comment]
+1

**Comment by hs0225** [comment]
+1

**Comment by zherczeg** [comment]
+1

**Comment by daeyeon** [comment]
+1

**Comment by chokobole** [comment]
+1

**Comment by yichoi** [comment]
Congratulations! I've added him to [here](https://github.com/Samsung/iotjs/wiki/Assigned-people)

**Comment by bzsolt** [comment]
Thank you! :)
